tools/compatibility-matrix-automation/turbo.json

```
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "compile": {
+      "outputs": ["dist/**", "lib/**"]
+    },
+    "test": {
+      "dependsOn": ["compile"]
+    }
```

There is no compile task, and test should probably be fine with the defaults

```suggestion
  "pipeline": {
    "test": {}
```